### PR TITLE
Open Source Project Initialization

### PR DESCRIPTION
**Pull request by AG3NTZ3R0**
A PR to set up the open source standards.

### CODE REVIEW

**Comment by AG3NTZ3R0** [comment]
The pylint workflow will continue to fail because the repository needs to be restructured by @jacob-bassi.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# JetBrains
+.idea
+
+# PyCache
+__pycache__
+
+# Python Development Environment
+build
+dist
+src/cnbc.egg-info
```
Comment (jacob-bassi) [inline .gitignore]: should this be 'src/yh-finance.egg-info' ?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+### Screenshots
+If applicable, add screenshots to help explain your problem.
+
+### Desktop Information
+ - OS: [e.g. MacOS]
+ - IDE [e.g. VSCode, PyCharm]
+ - Python Version [e.g. 3.12]
+ - Pip Version [e.g. 22]
+ - CNBC Version [e.g. 1.0.0]
```
Comment (jacob-bassi) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Change 'CNBC' to 'yh-finance'

**Review by jacob-bassi — CHANGES_REQUESTED** [review]
See comments on: 
- .github/ISSUE_TEMPLATE/bug_report.md
- .gitignore

File: .gitignore (hunk 1 above)
Comment (AG3NTZ3R0) [reply .gitignore]: Yes. I'll apply this in the next round of revisions.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (AG3NTZ3R0) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Yes. I'll apply this in the next round of revisions.

**Review by jacob-bassi — APPROVED** [review]

**Pull request merged**